LICENSE

```diff
@@ -0,0 +1,71 @@
+Apache License, Version 2.0
```

Comment:
Please check the license formatting. Please use the APACHE 2.0 LICENSE file that was shared.